types/react-color/lib/components/material/Material.d.ts

```diff
@@ -1,6 +1,22 @@
-import { Component } from "react";
+import { Component, CSSProperties } from "react";
 import { ColorPickerProps } from "../../..";
+import { Classes } from "reactcss";
 
-export type MaterialPickerProps = ColorPickerProps<MaterialPicker>;
+export interface MaterialPickerStylesProps {
+	material: CSSProperties;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/react-color/lib/components/material/Material.d.ts;
  Line: 6;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
I'm changing, but I haven't seen any .editorconfig style.